GlobalTestHelpers/Support/BitwardenTestCase.swift

```diff
@@ -9,10 +9,10 @@ open class BitwardenTestCase: XCTestCase {
 
     @MainActor
     override open class func setUp() {
-        if UIDevice.current.name != "iPhone 15 Pro" || UIDevice.current.systemVersion != "18.1" {
+        if UIDevice.current.name != "iPhone 16 Pro" || UIDevice.current.systemVersion != "18.1" {
```

Comment:
🤔 do you see a lint warning for this when you run locally?

Reply:
Weird, spacing got messed up somehow. That should be fixed now.